doc/manual/src/glossary.md

```diff
@@ -7,6 +7,59 @@
     translated into low-level *store derivations* (implicitly by
     `nix-env` and `nix-build`, or explicitly by `nix-instantiate`).
 
+    [derivation]: #gloss-derivation
+
+  - [store derivation]{#gloss-store-derivation}\
+    A [derivation] represented as a `.drv` file in the [store].
+    It has a [store path], like any [store object].
```

Comment:
What do you think about giving an example of `drv` path here? E.g. `/nix/store/0a2qvbq8b9xvrijp83p5m81xcw2xpgig-x509-validation-1.6.12.tar.gz.drv`

Reply:
Done.